ggml.c

```diff
@@ -11888,10 +11914,8 @@ GGML_CALL void ggml_rope_yarn_corr_dims(
     int n_dims, int n_orig_ctx, float freq_base, float beta_fast, float beta_slow, float dims[2]
 ) {
     // start and end correction dims
-    float start = floorf(ggml_rope_yarn_corr_dim(n_dims, n_orig_ctx, beta_fast, freq_base));
-    float end   =  ceilf(ggml_rope_yarn_corr_dim(n_dims, n_orig_ctx, beta_slow, freq_base));
-    dims[0] = MAX(0, start);
-    dims[1] = MIN(n_dims - 1, end);
+    dims[0] = MAX(0,         floorf(ggml_rope_yarn_corr_dim(n_dims, n_orig_ctx, beta_fast, freq_base)));
+    dims[1] = MIN(n_dims - 1, ceilf(ggml_rope_yarn_corr_dim(n_dims, n_orig_ctx, beta_slow, freq_base)));
 }
```

Comment:
It looks like you got out of sync with the state of master - these shouldn't be reverted.

Reply:
I've committed all the other changes but I'm a bit stumped on this one. Is this because master has moved on? What's the best way to merge these?  Should I be making a patch and creating a new PR based on the new master, or is there a slick way of doing this in the existing PR? Yesterday I think I was in the same boat and just did a new PR with everything re-merged. Sorry for my ignorance, I haven't worked with such a fast-moving project on GitHub before

Reply:
I think all changes have been made and the PR is ready for review

Reply:
I have fixed the errors in the last test. I also fixed a few related errors in the "examples" folder
All tests now pass in both make and cmake :

100% tests passed, 0 tests failed out of 22

Reply:
This change still needs to be reverted.

Reply:
I have now tracked down the correct version from master and re-committed in my branch